migration/template/upgrade-apps.md

```diff
@@ -0,0 +1,60 @@
+# Upgrade ${old_version}.x to ${new_version}.x
+
+## Prerequisites
+
+- [ ] Notify the users (if any) before the upgrade starts;
+- [ ] Check if there are any pending changes to the environment;
+- [ ] Check the state of the environment, pods, nodes and backup jobs:
+
+    ```bash
+    ./bin/ck8s test sc|wc
+    ./bin/ck8s ops kubectl sc get pods -A -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,READY-false:status.containerStatuses[*].ready | grep false
```

Comment:
```suggestion
    ./bin/ck8s ops kubectl sc|wc get pods -A -o custom-columns=NAMESPACE:metadata.namespace,POD:metadata.name,READY-false:status.containerStatuses[*].ready,REASON:status.containerStatuses[*].state.terminated.reason | grep false | grep -v Completed
```

Completed pods are ready=false in the yaml/json format, so this previous command would show all completed pods as not ready. This excludes the Completed pods and also shows the reason why any pods are not ready.